pkg/server/controller.go

```diff
@@ -81,6 +83,13 @@ type Notifier interface {
 }
 
 func New(c Config) (*Controller, error) {
+	if c.Configuration.BaseURL != "" {
+		_, err := url.Parse(c.Configuration.BaseURL)
+		if err != nil {
+			logrus.Error("base URL is invalid, some redirects might not work as expected")
```

Comment:
@petethepig Maybe you should return the err coz if you use that controller after this log this will lead to nil pointer error

Reply:
agreed, I changed it a bit. Thanks for the tip @Dzalevski !